lib/checkcondition.cpp

```diff
@@ -1010,14 +1010,14 @@ void CheckCondition::checkIncorrectLogicOperator()
                 incorrectLogicOperatorError(tok, text, alwaysTrue, inconclusive);
             } else if (printStyle && secondTrue) {
                 const std::string text = "If '" + cond1str + "', the comparison '" + cond2str +
-                                         "' is always " + (secondTrue ? "true" : "false") + ".";
+                                         "' is always true.";
```

Comment:
it seems Cppcheck should report this also. But I guess we dont check ternary operator conditions yet.

Reply:
There is now a ticket: http://trac.cppcheck.net/ticket/8254